File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    int towerHeight;
    do {
        towerHeight = get_int("How tall do you want the pyramid?: ");
    }
    while (towerHeight < 1 || towerHeight > 8);
    
    
    for (int row = 0; row < towerHeight; row++)
    {
        for (int dot = 0; dot < towerHeight - row - 1; dot++)
        {
            printf(" ");
        }
        for (int column = 0; column <= row; column++)
        {
            printf("#");
        }
        printf("\n");
    }
    
}

File: pset6/mario/mario.py
import cs50

while True:
    towerHeight = cs50.get_int("Pyramid Height?")
    if towerHeight < 1 or towerHeight > 8:
        towerHeight = cs50.get_int("Pyramid Height?")
    if towerHeight >= 1 or towerHeight <= 8:
        break

for i in range(towerHeight):
    print((towerHeight - i - 1) * " ", end = "")
    print((i + 1) * "#")
    